specs.bzl

```diff
@@ -94,9 +104,15 @@ def _maven_exclusion(group, artifact):
     # }
     return {"group": group, "artifact": artifact}
 
+def _maven_coord_str_to_unversioned(coord_str):
+    coord_split = coord_str.split(":")
+    return "{}:{}".format(coord_split[0], coord_split[1])
+
 maven = struct(
     repository = _maven_repository,
     artifact = _maven_artifact,
+    artifact_to_coord_str = _maven_artifact_to_coord_str,
```

Comment:
Can you inline these into the jetifier bzl file instead, since that's the only caller?

Reply:
Done.